packages/veritone-react-common/src/components/SearchBar/SearchBarContainer.js

```diff
@@ -490,7 +508,7 @@ class SearchBarContainer extends React.Component {
         ...customMenuActions,
       ]
     } else {
-      if (this.props.searchParameters && this.props.searchParameters.length > 0) {
+      if (this.props.searchParameters && this.props.isEditor && this.props.searchParameters.length > 0) {
         customMenuActions = [
           { label: 'Load Search Profile', onClick: (e) => { this.props.showLoadSavedSearch(e); this.handleMenuClose() } },
```

Comment:
They should be able to load profile, but not save.

Reply:
It will switch to else block. So I think it's will be same

Reply:
I see.